docroot/themes/humsci/su_humsci_admin/Gruntfile.js

```diff
@@ -102,13 +102,17 @@ module.exports = function(grunt) {
         precision: 10
       },
       dist: {
-        files: {
-          // Files are compiled individually so they may be included
-          // conditionally using logic built in to the theme template or module.
-
-          // BASE
-          'css/base/base.css':    'scss/base/base.scss',
-        }
+        files: [{
```

Comment:
Are you sure you want this change in here? This seems outside of the scope of the PR.

Reply:
yeah. allowed a dynamic scss compiling since the scss files that needed be modified were not compiling